### PR TITLE
[#13] feat: :sparkles: Adding Php 8.1 Support

### LINKED ISSUE #13: [toggle-crud] Add PHP 8.1 support
Epic: https://github.com/pheature-flags/pheature-flags/issues/321

Add support for PHP 8.1 in composer.json and test for possible incompatibilities.

### PR DESCRIPTION
**Pull request by AndreyPootMay**
Adding php ^8.1 support #hacktoberfest

Issue #13

---

The CI have an enviroment in 8.1?

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         }
     ],
     "require": {
-        "php": "^7.4|^8.0",
+        "php": "^7.4|^8.1",
```
Comment (kpicaza) [inline composer.json]: Hi @AndreyPootMay :wave: Thank you very much for the contribution :tada::tada::tada: 

Check the following suggestion, only allowing PHP ^7.4|^8.1 will block installing with PHP 8.0
```suggestion
        "php": "^7.4|^8.0|^8.1",
```

**Review by kpicaza — CHANGES_REQUESTED** [review]

**Comment by AndreyPootMay** [comment]
Check :D, greetings from México

File: composer.json.save [hunk 2]
```diff
@@ -0,0 +1,68 @@
+{
```
Comment (kpicaza) [inline composer.json.save:1]: This file is not need ;-D

**Review by kpicaza — CHANGES_REQUESTED** [review]
thanks again, check that you commit a backup of the composer.json file,  you can delete it and we will merge the PR.

**Comment by AndreyPootMay** [comment]
right, I check it

**Pull request closed without merging**